modules/pagination/client-react/containers/DataProvider.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect, useCallback } from 'react';
+
+import settings from '../../../../settings';
+
+const generateEdgesArray = quantity => {
+  const arr = [];
+  for (let i = 1; i <= quantity; i++) {
+    arr.push({ cursor: i, node: { id: i, title: 'Item ' + i } });
+  }
+  return arr;
+};
+
+const {
+  pagination: { web: limit, type }
```

Comment:
What if it is needed to use different data set sizes or types on other pages? How should we handle this?

Reply:
Fixed